sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -615,24 +615,56 @@ Once you have deployed KA Lite to a computer, there are a number of ways you can
 
-A more advanced way to customize your installation's behavior is by adding a file called `local_settings.py` in the `kalite` folder (the same folder that has `version.py`). Below is a list of options that you can set in `local_settings.py`, and what they mean.
-As of version 0.9.3 (Dec, 2012):
+Running KA Lite with your own settings
+______________________________________
+
+In a text editor, open up ``~/.kalite/settings.py`` (on Windows, locate ``C:\Users\<username>\.kalite``). That file is where you should put your custom settings, and KA Lite will load them automatically.
+
+You can also run the ``kalite`` with a completely different Python settings module by specifying ``kalite <command> --settings=my_settings_module``.
+
+
+Changing base settings
+^^^^^^^^^^^^^^^^^^^^^^
+
+By default, ``~/.kalite/settings.py`` will load ``kalite.project.settings.base`` which are the basic settings. But you can also load Raspberry Pi settings by changing the file to read something like:
+
+  .. ::
+    from kalite.project.settings.raspberry_pi import *
+    # Put your settings here, e.g.
+    # MY_SETTING_VAR = 123
 
+
+Available settings
+__________________
+
+
+Most common settings
+^^^^^^^^^^^^^^^^^^^^
+
+* DEBUG = <True or False> (default = False) -- Enables debug mode. In case you run into technical issues, enable this setting before troubleshooting / reporting.
 * CONTENT_ROOT = "<path to desired content folder>" (default=ka-lite/content)
-  This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory.
+  This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.
 * TIME_ZONE = <desired time zone>  (default = "America/Los_Angeles")
   You can set this to be the local time zone for your installation. Choices can be found here.
 * LANGUAGE_CODE = "<desired ISO 639-1 Language Code>" (default = "en-us")
   You can set this to the desired language code for this installation (All choices can be found here).  If there are translations available, our web server will show them in KA Lite. Soon, we hope to provide support for internationalized content inside the KA Lite interface.
 * USE_I18N = <True or False> (default = True)
   If you set this to False, our web server will make some optimizations so as to avoid loading internationalization tools. Things might run a little faster, but you won't have support for translated content.
-* USE_L10N = True
+* USE_L10N = <True or False> (default = True)
   By default, this is set to False. If you set this to True, Django will format dates, numbers and calendars according to the current locale. For example, January 5, 2000 would be 1/5/2000 if locale = "en-us" and 5/1/2000 if locale = "en-gb"
```

Comment:
Hmm, yes, I don't know what's going on here... default is `False`... so I'll put that!